### PR TITLE
FSET-684: Add DOB, Address and post code to the successful candidates report

### PR DESCRIPTION
**Pull request by henricook**

### CODE REVIEW

File: app/controllers/ReportingController.scala [hunk 1]
```diff
@@ -191,7 +191,7 @@ trait ReportingController extends BaseController {
           a <- applications
           t <- testResults.get(a.applicationId)
           c <- cDetails.get(a.userId)
-        } yield PassMarkReportWithPersonalData(a, t, PhoneAndEmail(c.phone, Some(c.email)))
+        } yield PassMarkReportWithPersonalData(a, t, ContactDetails(c.phone, c.email, c.address, c.postCode))
```
Comment (henricook) [inline app/controllers/ReportingController.scala]: Notes: This leads to additional data being returned for the pass mark report that it doesn't use, but it's a sensible exchange object reuse IMHO

File: conf/application.conf [hunk 2]
```diff
@@ -218,4 +218,4 @@ mongodb {
   uri = "mongodb://localhost:27017/fset-fasttrack"
 }
 
-maxNumberOfDocuments = 10
\ No newline at end of file
+maxNumberOfDocuments = 20000
```
Comment (henricook) [inline conf/application.conf]: Notes: This was a leftover from an old config migration, the 'test' value of 10 made it into the main application.conf! The test value was moved to test parameters, the original value was 10,000 but on consultation we've made this 20,000 for safety

**Comment by henricook** [comment]
Tests are failing because of a mongo error

File: app/controllers/ReportingController.scala (hunk 1 above)
Comment (ollyw) [inline app/controllers/ReportingController.scala]: I concur!

File: app/model/Commands.scala [hunk 3]
```diff
@@ -203,7 +204,7 @@ object Commands {
     onlineTestPassmarkEvaluations: OnlineTestPassmarkEvaluationSchemes)
 
   case class PersonalInfo(firstName: Option[String], lastName: Option[String], preferredName: Option[String],
-    aLevel: Option[String], stemLevel: Option[String])
+    aLevel: Option[String], stemLevel: Option[String], dateOfBirth: Option[String])
```
Comment (ollyw) [inline app/model/Commands.scala]: Perhaps the type for DOB should be Option[LocalDate]?

**Pull request merged**